spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2184Test.java

```diff
@@ -1,29 +1,21 @@
 package edu.umd.cs.findbugs.detect;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnJre;
+import org.junit.jupiter.api.condition.JRE;
 
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
 import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
 import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
 
-import static org.junit.Assume.assumeFalse;
-import static org.junit.Assume.assumeThat;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 
-public class Issue2184Test extends AbstractIntegrationTest {
-    @Before
-    public void verifyJavaVersion() {
-        assumeFalse(System.getProperty("java.specification.version").startsWith("1."));
-        int javaVersion = Integer.parseInt(System.getProperty("java.specification.version"));
-        assumeThat(javaVersion, is(greaterThanOrEqualTo(14)));
-    }
+class Issue2184Test extends AbstractIntegrationTest {
 
     @Test
-    public void test() {
+    @DisabledOnJre(JRE.JAVA_11)
```

Comment:
See line 19 below, these are for higher jdks (yes originally 14) but we don't use 14 to build as its obsolete.  Instead we have certain sets of tests for the jdks we use to build with.  The reason this even worked as-is results from fact repo only used jdk 17 to build.  Once my other PR is applied, this would in fact fail if jdk 11 was still allowed for consideration.

Reply:
And note, that also means with removal of toolchains.  On Github, toolchains won't work for a few reasons.  Mainly because there was no request to download other jdks to start with.  Its toolchained to itself currently  In my other PR I remove that for now as its not appropriate to demonstrate what we are trying to show and if it were, it would need more in GHA to download other jdks to start with.  The overhead of downloading a bunch of jdks isn't too worth the effort when really just trying to show users its as compatible as possible as we can here without going totally overboard.

Reply:
Then only the test samples should be built with different JDKs, and the tests should be run with different JREs, not the entire project. We don't need to show that the SpotBugs project can be built using Java 8, but that it can analyze code built with Java 8, and can be run by Java 11.